requirements.txt

```diff
@@ -29,3 +29,6 @@ pymongo==4.7.2
 
 # Handling for Datadog
 ddtrace~=2.21.3
+
+# BI Data Collection
+confluent-kafka
```

Comment:
@kevklash , Let's pin this at minimum to a major version so it doesn't break in the future.